blitz/blitz_generator.go

```diff
@@ -623,7 +676,54 @@ var templates = []SandboxConfigTemplate{
 		},
 	},
 	{
-		// Lots of yellow BAFs, player can't shoot, have to get score boxes
+		// Rolling spheres and rolling cubes, motherships, optional bafs, symbiose mode
+		name: "Multiple enemies vs Plasma",
```

Comment:
FYI: symbiosis mode is currently buggy in the sandbox, but it will be fixed next update.

The good news is that because I introduced a breaking change, I won't be able to update the Blitz server before the next update, so those new configs can stay.